WcaOnRails/spec/controllers/incidents_controller_spec.rb

```diff
@@ -0,0 +1,292 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe IncidentsController, type: :controller do
```

Comment:
Did you know that Controller specs are deprecated in Rails 5 (see https://blog.bigbinary.com/2016/04/19/changes-to-test-controllers-in-rails-5.html)? We really shouldn't be writing any more of them, and in fact should be working to get rid of our existing ones. @jonatanklosko, do you have any ideas for maybe preventing people from writing new controller specs?

For this specific test file, I suppose we can live with it as is. I'm just concerned that at some point, we're going to be prevented from upgrading to rails 6 because controller specs no longer work with them.

Reply:
Well, what I can think of is having a special test case that could do some checks, e.g. if the number of `spec/controllers/**_spec.rb` files is bigger than we would expect (like a comparison to some past commit, whatever). This wouldn't prevent one from adding more test cases to existing files though, to achieve so we would need to do more sophisticated stuff, like counting `it`s or comparing content hash digest, but both of these seem to be an overkill IMO. Also it may even make more sense to spend the time on rewriting a spec or two rather than figuring out a good detection system ^^

Reply:
I agree. Probably better to spend our time porting controller specs to request specs :)

Reply:
I knew nothing about that, I've reworked the controller spec to be a requests spec.
Although I did not encounter any issue using `assigns` or `render_template`, I've gotten rid of them, and included some additional equivalent tests in the feature spec.

Please let me know if I did this right!